src/components/form/tickets/index.tsx

```diff
@@ -0,0 +1,138 @@
+import { AppDispatch } from '../../../store';
+import CustomSelect from './select';
+import DatePicker from './datePicker';
+import FormInput from '../../common/formInput';
+import TicketService from '../../../service/TicketService';
+import { createTicketAsync } from '../../../store/tickets/thunk';
+import { updateTicket } from '../../../store/tickets/slice';
+import { Controller, SubmitHandler, useForm } from 'react-hook-form';
+import {
+  EmptyHelperText,
+  FlexContainer,
+  HelperText,
+  StyledCancelButton,
+  StyledHeading,
+  StyledLoginButton,
+} from './styled';
+import { ITicketFieldValues, TicketsFormHelper, statusOptions } from './helper';
+import { ITicketState, ITickets, IUpdateTickets } from '../../../types/tickets';
+import { memo, useEffect } from 'react';
+
+interface IProps {
+  toggleModal: () => void;
+  refetchTickets: () => void;
+  initialValues: ITicketState | null;
+  dispatch: AppDispatch;
+  isEdit?: boolean;
+}
+const TicketsForm = ({ toggleModal, refetchTickets, initialValues, dispatch, isEdit }: IProps) => {
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+    setValue,
+    control,
+  } = useForm<ITicketFieldValues>();
+
+  useEffect(() => {
```

Comment:
Use defaultValues instead of using `setValue` in useEffect.

Reply:
fix: fixed by patch